pop-cli-for-appchains/guides/launch-a-chain/launch-a-known-chain.md

```diff
@@ -0,0 +1,71 @@
+# Launch a Known Chain
+
+With Pop CLI you can quickly spin up supported parachains without needing to prepare network configuration files.
+
+A `supported parachain` currently refers to one of the system parachains or `pop`. Pop CLI automatically fetches the required binaries and chain-spec generators so you can launch in seconds. Support for additional parachains may be added in the future.
+
+### Example Usage
+
+Spin up Paseo with the Asset Hub and Passet Hub parachains:
+
+```shell
+pop up paseo -p asset-hub,passet-hub
+```
+
+Run Kusama on port `8833` with Asset Hub parachain assigned to `9944`:
+
+```shell
+pop up kusama --port 8833 --parachain asset-hub:9944 -r stable2412-4
+```
+
+Launch Polkadot with two parachains, including a specific ParaId for Pop:
+
+```shell
+pop up polkadot --port 8833 --parachain asset-hub:9977,pop#3395:9944 -r stable2412-4
+```
+
+### Custom Network Configurations
+You can still provide a full network configuration file if needed. The `pop up network` command now accepts a positional path argument, eliminating the need for `--file`/`-f`:
+
+```bash
+touch paseo-local.toml
+```
+
+```toml
+[relaychain]
+chain = "paseo-local"
+
+[relaychain.genesis_overrides.sudo]
+key = "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY" # Alice
+
+[[relaychain.nodes]]
+name = "alice"
+rpc_port = 57731
+validator = true
+
+[[relaychain.nodes]]
+name = "bob"
+validator = true
+
+[[relaychain.nodes]]
+name = "charlie"
+validator = true
+```
+
+As you can see, the sudo account (admin of the chain) is overridden with `Alice` account. This allows us to make changes to Paseo Local if needed.
```

Comment:
A bit of a nit pick, but I think going in trying to explain pallet_sudo and how the zombienet config works might require more than we should provide here.

I'd rather do one of the following:

Remove this sentence and point users to learn more about the zombienet config files pointing them to the zombienet docs.

Or rephrase to something like:
"Note that we are using `genesis_overrides` in the above config to set a concrete value to the `sudo` key at genesis. This concrete key in the example is specific to `pallet_sudo` and we could only overwrite it as long as this concrete pallet is present in this chain's runtime.

Learn more about what this pallet does at:` <link to the pallet source>`
Learn more about creating this type of configuration files at: `<link to zombienet spec>`
"